SmartDeviceLink/SDLPreloadChoicesOperation.m

```diff
@@ -87,12 +87,10 @@ - (void)sdl_preloadCellArtworksWithCompletionHandler:(void(^)(NSError *_Nullable
 
     NSMutableArray<SDLArtwork *> *artworksToUpload = [NSMutableArray arrayWithCapacity:self.cellsToUpload.count];
     for (SDLChoiceCell *cell in self.cellsToUpload) {
-        if ([self.windowCapability hasImageFieldOfName:SDLImageFieldNameChoiceImage]
-            && [self sdl_artworkNeedsUpload:cell.artwork]) {
+        if ([self sdl_shouldSendChoicePrimaryImage] && [self sdl_artworkNeedsUpload:cell.artwork]) {
```

Comment:
`sdl_shouldSendChoicePrimaryImage` and `sdl_shouldSendChoiceSecondaryImage` default to `YES` if `windowCapability` is `nil` so the images still get uploaded/displayed.

Reply:
`windowCapability` cannot be `nil`

Reply:
I guess I am confused because the title for this PR is "Fix PreloadChoicesOperation uploading images when WindowCapability is nil"

Reply:
Fixed the title

Reply:
Just so we are on the same page, if `windowCapability.imageFields` is `nil`, you do not want to upload the choice set images, right? Because right now they are still uploaded if `windowCapability.imageFields` is `nil`.

Reply:
The intention is that if it’s nil, it is uploaded. The main PR for this conversation was a few weeks ago on #1575

Reply:
Ok. Can you add documentation to the methods because the description on the issue and on this PR make it sound that you don't want to upload images when the properties are `nil`.